web/js/map/natural-events/ui.js

```diff
@@ -314,7 +309,7 @@ export default function naturalEventsUI(ui, config, store, models) {
       self.markers = naturalEventMarkers.draw();
       naturalEventsTrack.update(null);
     }
-    store.dispatch(deselectEventAction);
+    // store.dispatch(deselectEventAction);
```

Comment:
This dispatch does seem weird and looks like it could cause some sort of never ending dispatch loop but removing it seems to also introduce an error where an event can be active in the "map" state but not in the "Global" state.

to reproduce:
1. Click on event in middle of map.
2. change to polar proj.
3. return to geo proj.
4. see error : Event is selected in list but not on map

Reply:
Hmmm, yeah this seemed like either a) it wasn't doing anything since `deselectEvent` what is being called when a deselect event action is dispatched or b) was causing an infinite loop of dispatching.

With this line still commented out, I wasn't able to reproduce this specific error with the few events I tried.  Is there a specific event that is breaking this for you?

Reply:
I'm not seeing the error anymore. 🤯